Cargo.toml

```diff
@@ -38,7 +38,7 @@ hmac = "0.12.1"
 jwt-simple = { version = "0.12", default-features = false, features = [
     "pure-rust",
 ] }
-kbs-types = { "git" = "https://github.com/virtee/kbs-types.git", rev = "e3cc706" }
+kbs-types = { "git" = "https://github.com/Xynnn007/kbs-types.git", rev = "718a83a" }
```

Comment:
I will change this once https://github.com/virtee/kbs-types/pull/69 gets merged.